Source/CBLRestPusher.m

```diff
@@ -560,7 +560,10 @@ - (BOOL) uploadMultipartRevision: (CBL_Revision*)rev {
                           _dontSendMultipart = YES;
                           [self uploadJSONRevision: rev];
                       } else {
-                          self.error = error;
+                          // 403/Forbidden means validation failed; don't treat it as an error
+                          // because I did my job in sending the revision.
+                          if (error.code != kCBLStatusForbidden)
```

Comment:
Both uploadMultipartRevision: and uploadJSONRevision: are check the error from the request directly which are different from uploadBulkDocs:changes: that checks the error from the result. I'm not sure if we should treat kCBLStatusUnauthorized as not an replication error.